### DISCUSSION TITLE
Add support of nc and nci files

### DISCUSSION THREAD

**Discussion by thmbc** [sbarex/SourceCodeSyntaxHighlight#275] [Q&A]
Hello,
Can you add support of nc and nci files in G code (language to pilot numerical command machines)?
Do not need to have syntax highlight but to see it in QuickLook.
nc and nci files are plain text files.

Thanks a lot.
Best Regards
Thomas

**Comment by thmbc** [comment]
Hello,
can you add support of NC and NCI files as plain text files ?

output by mdls command : 
kMDItemContentType     = "dyn.ah62d4rv4ge80622"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80622",
    "public.data"

![image](https://github.com/sbarex/SourceCodeSyntaxHighlight/assets/169804663/3a2e8086-a111-4e40-8fb0-eef7876f1073)

Thanks a lot
Thomas

**Reply by Andrew-Barton** [reply]
+1 for adding .nc g-code files.
There is a VS Code extension for syntax highlighting here:
[https://github.com/Milys/NC-Gcode-VSCode](url)
Thank you

**Comment by sbarex** [comment]
Implemented on release 2.1.27 (76).
